Cargo.toml

```diff
@@ -184,6 +184,7 @@ raft = { git = "https://github.com/tikv/raft-rs", branch = "master", default-fea
 raft-proto = { git = "https://github.com/tikv/raft-rs", branch = "master", default-features = false }
 protobuf = { git = "https://github.com/pingcap/rust-protobuf", rev = "65e9df20fbcbcf2409d5ee86a2332ecd04c534f8" }
 protobuf-codegen = { git = "https://github.com/pingcap/rust-protobuf", rev = "65e9df20fbcbcf2409d5ee86a2332ecd04c534f8" }
+grpcio = { git = "https://github.com/hunterlxt/grpc-rs", branch = "xt/add-before-executor-api", default-features = false, features = ["openssl-vendored"] }
```

Comment:
This needs to be merged first.

Reply:
Let's push https://github.com/tikv/grpc-rs/pull/502 to be merged.